auxin_cli/src/state.rs

```diff
@@ -716,9 +582,18 @@ impl AuxinStateManager for StateManager {
 						//} else {
 
 						let bytes = s.serialize()?;
-						file.write_all(bytes.as_slice())?;
-						file.flush()?;
-						drop(file);
+						let mut file = OpenOptions::new()
+							.truncate(true)
```

Comment:
Might make sense to remove truncate here?

Reply:
Yeah, wont actually change behavior though since we error if the file already exists above this.